client/dataset_test.go

```diff
@@ -12,10 +12,14 @@ func TestDatasets(t *testing.T) {
 
 	c := newTestClient(t)
 	datasetName := testDataset(t)
+	datasetDescription := ""
+	datasetExpandJSONDepth := 0
 
 	currentDataset := &Dataset{
-		Name: datasetName,
-		Slug: urlEncodeDataset(datasetName),
+		Name:            datasetName,
+		Description:     &datasetDescription,
```

Comment:
I think _not_ setting these here proves that they are truly not required and that the default values work as intended